src/locales/en_US.js

```diff
@@ -49,10 +49,22 @@ const strings = {
   alert_dropdown_alert: 'Alert! ',
   alert_dropdown_warning: 'Warning! ',
 
+  and: 'and',
```

Comment:
This should use string replacement. Also use an easier to search key than `and`

`this_and_that: '%1$s and %2$s'`

Reply:
This is used for a string list of potentially 3+ elements, not just two elements. Is this pattern safe translation-wise for 3+ elements?